docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,501 @@
+    "\n",
+    "All of these examples has low label quality scores to reflect their low annotation quality.\n",
+    "\n",
+    "### Other uses of visualize function\n",
+    "The visualize function is also equally able to visualize non-issues, labels, predictions or just the image seperately and save the created images. Lets take a look at the first couple of images in our dataset with this functionality.\n",
+    "\n",
+    "We can save the image by providing a `save_path` variable. Notice how the label quality score is high for this example and it is marked as a non issue. Both the given and predicted labels closeley resemble each other contributing to the high score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e770d23",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "idx = 0\n",
+    "image_path = IMAGE_PATH + labels[idx]['seg_map']\n",
+    "print(image_path, '| idx', idx , '| label quality score:', scores[idx], '| is issue:', is_label_issue[idx])\n",
+    "visualize(image_path, label=labels[idx], prediction=predictions[idx], class_names=class_names, save_path='./example_image.png')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c9464e8",
+   "metadata": {},
+   "source": [
+    "For the next example, notice how we are only passing in the given labels to visualize. We can limit visualization to either labels, predictions or neither."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "57e84a27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "idx = 1\n",
+    "image_path = IMAGE_PATH + labels[idx]['seg_map']\n",
+    "print(image_path, '| idx', idx , '| label quality score:', scores[idx], '| is issue:', is_label_issue[idx])\n",
+    "visualize(image_path, label=labels[idx], class_names=class_names)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8744ab9",
+   "metadata": {},
+   "source": [
+    "Finally, we can also just look at the image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0302818a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "idx = 2\n",
+    "image_path = IMAGE_PATH + labels[idx]['seg_map']\n",
+    "visualize(image_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ce74938",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "import numpy as np\n",
+    "\n",
+    "assert set([60, 48, 1, 46, 30]).issubset(issue_idx)\n",
```

Comment:
```suggestion
    "assert issue_idx[issue_to_visualize] == 32\n",
    "assert set([60, 48, 1, 46, 30]).issubset(issue_idx)\n",
```

Reply:
I think your test was not correct before as it was checking `issue_idx` but the code cell is only showing the first few elements of these, so all the indices need to be amongst these first few elements.